sdk/monitor/monitor-query/src/internal/modelConverters.ts

```diff
@@ -142,16 +142,17 @@ export function fixInvalidBatchQueryResponse(
     // the body here is incorrect, deserialize the correct one from the raw response itself.
 
     // deserialize the raw response from the service, since we'll need index into it.
-    if (!wholeResponse) {
-      wholeResponse = JSON.parse(
-        generatedResponse["_response"].bodyAsText
-      ) as GeneratedQueryBatchResponse;
-    }
+
```

Comment:
you can still access that `bodyAsText` using the `onResponse` option in `OperationOptions`, see https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/schemaregistry/schema-registry/src/schemaRegistryClient.ts#L148-L162